datasets/web_of_science/web_of_science.py

```diff
@@ -0,0 +1,150 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors and the HuggingFace NLP Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""Web of science"""
+
+from __future__ import absolute_import, division, print_function
+
+import os
+
+import nlp
+
+
+_CITATION = """\
+@inproceedings{kowsari2017HDLTex,
+title={HDLTex: Hierarchical Deep Learning for Text Classification},
+author={Kowsari, Kamran and Brown, Donald E and Heidarysafa, Mojtaba and Jafari Meimandi, Kiana and and Gerber, Matthew S and Barnes, Laura E},
+booktitle={Machine Learning and Applications (ICMLA), 2017 16th IEEE International Conference on},
+year={2017},
+organization={IEEE}
+}
+"""
+
+_DESCRIPTION = """\
```

Comment:
Could you add a general description of the dataset ? For example what kind of classification this dataset is for